test/details_page_test.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:contacts_data/ui/details_page.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:network_image_mock/network_image_mock.dart';
+
+void main() {
+  testWidgets('Check for existence of widgets', (WidgetTester tester) async {
+    await mockNetworkImagesFor(
+      () => tester.pumpWidget(const MaterialApp(
+        home: DetailsPage(
```

Comment:
Remember that you have a json test file for this kind of data

Reply:
done